### DISCUSSION TITLE
The example code produces an NullReferenceException

### DISCUSSION THREAD

**Discussion by peckebrecht** [stephanstapel/ZUGFeRD-csharp#505] [General]
I used the example code from the README in an .NET 8.0 test project and it produces an NullReferenceException on Save. I can't tell what's null here.

 System.NullReferenceException: Object reference not set to an instance of an object.

  Stack Trace: 
InvoiceDescriptor23CIIWriter.Save(InvoiceDescriptor descriptor, Stream stream, ZUGFeRDFormats format)
InvoiceDescriptor23Writer.Save(InvoiceDescriptor descriptor, Stream stream, ZUGFeRDFormats format)
InvoiceDescriptor.Save(Stream stream, ZUGFeRDVersion version, Profile profile, ZUGFeRDFormats format)

**Comment by stephanstapel** [comment]
Do you know in which line the exception is thrown?

**Comment by peckebrecht** [comment]
No.

**Comment by stephanstapel** [comment]
I am afraid that I can't help if you can't provide additional information

**Comment by peckebrecht** [comment]
I just wanted to help with a bug report.
